Ginger/GingerCoreCommonTest/VariableTests/SelectionListVariableTests.cs

```diff
@@ -526,6 +526,64 @@ public void SelectionListVar_TestDeleteAllValues()
             string setErrorMsgExpectedResult = string.Empty;
 
 
+            //Assert
+            Assert.AreEqual(setValueExpectedResult, setValueResult, "Set Value Result");
+            Assert.AreEqual(setErrorMsgExpectedResult, errorMsg, "Error Msg Result");
+        }
+
+        [TestMethod]
+        [Timeout(60000)]
+        public void SelectionListVar_TestGetLength()
+        {
+            //Arrange
+            VariableSelectionList variableSelectionList = new VariableSelectionList();
+            OptionalValue newVal1 = new OptionalValue("One");
+            variableSelectionList.OptionalValuesList.Add(newVal1);
+            OptionalValue newVal2 = new OptionalValue("Two");
+            variableSelectionList.OptionalValuesList.Add(newVal2);
+
+            Dictionary<string, string> extraParamsDict = new Dictionary<string, string>();
```

Comment:
there right and full way to test it is to create in the unit test expression of selection list variable with the GetLength param in it and then run the VE calculation and expect to get the length value, because you want also to test all VE "Calculate" process